{"name": "Test login with invalid username", "status": "passed", "description": "This test verifies that an error message is displayed when logging in with an invalid username.", "attachments": [{"name": "Invalid Username - Error Message", "source": "c07a1912-197d-434a-a2c7-d9501195509c-attachment.png", "type": "image/png"}], "start": 1740510321697, "stop": 1740510324441, "uuid": "e9ff1400-e69a-4026-91f7-0d9f42499d8b", "historyId": "8dfefa9403d2f0167e5ccac7cf061037", "testCaseId": "8dfefa9403d2f0167e5ccac7cf061037", "fullName": "test_form_authentication.TestFormAuthentication#test_invalid_username", "labels": [{"name": "feature", "value": "Form Authentication"}, {"name": "story", "value": "Invalid Login"}, {"name": "suite", "value": "test_form_authentication"}, {"name": "subSuite", "value": "TestFormAuthentication"}, {"name": "host", "value": "Jonathan"}, {"name": "thread", "value": "22196-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_form_authentication"}]}